ci/vm.sh

```diff
@@ -59,22 +59,19 @@ make_disk() {
 }
 
 gen_metadata() {
-	local cprcmd hardware_id id
-	id=$(uuidgen)
-	hardware_id=$(uuidgen)
-	local class=$1
-	local slug=$2
-	local tag=$3
-	local distro=${slug%%_*}
-	local version=${slug#*_}
-	version=${version//_/.}
+	local class=$1 slug=$2 tag=$3 id=$4
 
+	local cprcmd
 	cprcmd=(cat)
-
 	if [[ $UEFI == true ]] && [[ $arch != aarch64 ]]; then
 		cprcmd=(jq -S '.filesystems += [{"mount":{"create":{"options": ["32", "-n", "EFI"]},"device":"/dev/sda1","format":"vfat","point":"/boot/efi"}}]')
 	fi
 
+	local distro=${slug%%_*} version=${slug#*_}
```

Comment:
optional nit: prefer declaration and assignment as separate statements. 

There is no behavioral difference here, but it prevents awkward behaviors when used with command substitution; as the local builtin does not propagate the exit code from the command substitution.

Reference: https://google.github.io/styleguide/shellguide.html#use-local-variables

(feel free to disregard this comment as it seems to disagree with this code base)

Reply:
Shellcheck errors on the command substitution case. I tend to do the obvious/simplest/least-lines case unless a tool complains. In this case:
```bash
local foo=1 bar=2
```
seems more obvious than:
```bash
local foo bar
foo=1
bar=2
```
The latter would have me wondering why its separated. I think for shellscripts its kind of weird as most uses are interactive and the latter case (separate declaration and definition) is something anyone will do at a command prompt.

Reply:
^ is *not* something anyone would do at a prompt